cli/gfuzz/commands/gen/cpp/gen_cpp.py

```diff
@@ -53,7 +53,7 @@
 '''
 
 CPP_SAVE_REF = '''\
-    out_ref[{out_ref_idx}] = reinterpret_cast<void *>({arg_name});
+    out_ref[{out_ref_idx}] = (void*)({arg_name});
```

Comment:
I think there was a reason I used `reinterpret_cast` here over C-style casts, bit I can't remember the specifics. Can you explain the rationale for this change?

Reply:
I had troubles with harnessing a C library unless I used this.